### DISCUSSION TITLE
Which rationale justify that the efficiency of the reversible engine is a function of the temperatures, $e(t_1, t_2)$?

### DISCUSSION THREAD

**Discussion by josePereiro** [josePereiro/TankeFactory#2] [Discussions]
I’m struggling to clearly link Carnot’s heuristic premises (e.g. the non-existence of perpetual motion, spontaneous heat sense, etc.) with the exclusive dependency of the efficiency of the reversible engine on the reservoirs’ temperature.

My current rationale is as follows: I think it is depend on the particular definition of a heat engine used. You can't derived it directly from the mentioned heuristics. Once you define an engine as a flow of heat (caloric version) across two reservoirs with possible resulting work and impose an upper limit on efficiency (and the 1st law), the only free parameters will be the temperatures.

But,  actually, why is it obvious that temperature plays a role at all? Any thoughts?

**Comment by ta-rekt** [comment]
Yes exactly. I think that's what they do. You have to define an engine in some way to argue about all this. I think that definition of an engine you gave was part of Carnot's premises.

I'll try to summarize what I understood. If you have any engine, reversible or not, which has a greater efficiency than some reversible engine M for the same temperatures, then you can create a perpetual motion machine by connecting both of them to the same two reservoirs. Assuming that doesn't exist, there can be no engine with a greater efficiency than that reversible engine M we just considered, at those temperatures. 

Therefore M is the most efficient engine possible at these temperatures, and the only thing we assumed about M in this argument is that it is some reversible engine, with no other requirements. Therefore it applies just as well to any reversible engine. In other words all reversible engines are _the_ most efficient possible engine, so they must all have the same efficiency at a given pair of temperatures.

If you define an engine as:

- something that functions using two reservoirs at different temperatures t_1 and t_2
- generates work w and dissipates heat q

and nothing else, then your engine's efficiency (or any other variable) a priori should be a function of these 4 parameters: $e(w, q, t_1, t_2)$. The argument above says that if the engine is reversible and the temperatures are held constant, then w and q don't affect the efficiency anymore because it is constant, so you can remove them from the function, and you end up with $e(t_1, t_2)$.

Why is temperature playing a role at all? I think this is an assumption they use from experience. I mean, temperature wasn't even well defined at the time Carnot thought about all this.

**Reply by josePereiro** [reply]
The ‘No Motion Machine’ is indeed a stronger constraint on the universe than one might initially expect. This same analysis can be applied to any form of energy. Any transformation from energy to work and vice versa will have such a limit, which opens the door to thermodynamic analysis in any macroscopic system, such as in chemistry.

**Reply by ta-rekt** [reply]
That's a great point. But I guess heat dissipation is common to all conversions from one energy form to another, so there should be elements from this analysis in common with all the others, including the existence of temperature, i guess?

**Reply by josePereiro** [reply]
But, I think is more general, you don't need to talk about neither heat nor temperature. No even about physics. Any transformation of a conserved quantity into different forms, if one of this forms is not observed at equilibrium ($t \to \infty$), this imply that the transient transformations into the missing form are less efficient in the forward direction (the equality being useful only for modeling reasons), and that there is other form which is produced with perfect efficiency.  

Yes, in the case of thermodynamics (pre statistical physics), the missing energy form at equilibrium is the kinetic energy of macroscopic chunks of matter, so the ineffitien energy transfer is work, and the perfect form of energy transfer is heat (a machine producing heat with efficiency $1$ is easy, at least from work).
